### PR TITLE
boards: arm: lora_e5_mini: new board support

### PR DESCRIPTION
**Pull request by mniestroj**
This board is similar to [lora_e5_dev_board](https://docs.zephyrproject.org/3.5.0/boards/arm/lora_e5_dev_board/doc/lora_e5_dev_board.html), but with smaller form-factor, which makes it better suited as low volume prototype device.

### CODE REVIEW

**Review by bbilas — DISMISSED** [review]

**Comment by mniestroj** [comment]
@erwango Could you take a look? Thanks!

File: boards/arm/lora_e5_mini/lora_e5_mini.dts [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+	aliases {
+		led0 = &red_led_1;
+		sw0 = &boot_button;
+		lora0 = &lora;
+		watchdog0 = &iwdg;
+	};
+};
+
+&lptim1 {
```
Comment (erwango) [inline boards/arm/lora_e5_mini/lora_e5_mini.dts:48]: stm32_lp_tick_source: &lptim1 {

**Review by erwango — CHANGES_REQUESTED** [review]
Otherwise LGTM

File: boards/arm/lora_e5_mini/lora_e5_mini.dts (hunk 1 above)
Comment (mniestroj) [reply boards/arm/lora_e5_mini/lora_e5_mini.dts]: Applied!

**Review by bbilas — APPROVED** [review]

**Review by erwango — APPROVED** [review]

**Pull request merged**